crates/runestick/src/modules/tests.rs

```diff
@@ -0,0 +1,45 @@
+//! Utilities related to testing
```

Comment:
Since this adds a couple of dependencies and is currently only used for testing, could we move it to `rune-tests` instead?

Reply:
Yes, sure!